src/style/app.scss

```diff
@@ -226,9 +226,14 @@ select::-ms-expand {
 .markdown ul, .markdown p {
   margin-bottom: 25px;
 }
-
 #fileInputWrapper{
   position:absolute;
   width:50%;
   height:41px;
 }
+
+.show-condition-gray-bar {
```

Comment:
Couldn't the Gov.UK class for the gray bar be re-used instead of introducing a new class?

Reply:
Done.